src/api/app/models/bs_request_action.rb

```diff
@@ -238,7 +238,7 @@ def notify_params(ret = {})
   end
 
   def contains_change?
-    return sourcediff.present?
+    return sourcediff(nodiff: 1).present?
```

Comment:
What does `nodiff` do? If this suppresses the diff the `.present?` will not return the opposite of what it returned before.